SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaAndSnapshotsCommandlet.h

```diff
@@ -20,8 +20,15 @@ class UGenerateSchemaAndSnapshotsCommandlet : public UCommandlet
 	virtual int32 Main(const FString& Params) override;
 
 private:
-	void GenerateSchema(FSpatialGDKEditor& SpatialGDKEditor);
-	void GenerateSnapshots(FSpatialGDKEditor& SpatialGDKEditor);
-	TArray<FString> GetAllMapPaths(FString InMapsPath);
-	void GenerateSnapshotForMap(FSpatialGDKEditor& SpatialGDKEditor, FString WorldPath);
+	const FString MapPathsParamName = TEXT("MapPaths");	//Commandline Argument Name used to declare the paths to generate schema/snapshots against
+	const FString GameDirName = TEXT("/Game");			//Root directory name that maps will ultimately be found in
```

Comment:
This will be project specific. I believe you should use FPath::ProjectDir()

Reply:
Fabulous! I didn't know a project could configure itself not to use the /Game dir?

Reply:
Yep, can be configured to anything I believe.

Reply:
I believe this may be a misunderstanding. `FPath::ProjectDir` will return the name of the _directory_ containing the game.
`/Game` that is used here is the beginning of the _asset path_ used in Unreal's asset resolution.
Regardless of the name of the project, the asset paths will still begin with `/Game`, e.g. even if my map lives in `/ShooterGame/Content/Maps/Highrise.umap`, the asset path will still be `/Game/Maps/Highrise`.

Reply:
Alright, then I shall resolve this as "no change".